### PR TITLE
updated default folders locations

### PR DESCRIPTION
**Pull request by SamMarkowitz**

### CODE REVIEW

File: cloudslang_cli.rst [hunk 1]
```diff
@@ -208,11 +208,10 @@ system properties files are being used and they contain a system
 property with the same fully qualified name, the property in the file
 that is loaded last will overwrite the others with the same name.
 
-System property files can be loaded automatically if placed in a folder
-named ``properties`` in the CLI's **bin** directory. If the
-flow or operation requires a system properties file that is not loaded
-automatically, use the ``--spf`` flag and a comma-separated list of file
-paths.
+System property files can be loaded automatically if placed in a folder located
+at ``cslang/configuration/properties``. If the flow or operation requires a
```
Comment (levice14) [inline cloudslang_cli.rst]: for properties files the search is recursive - meaning all the SP files will be loaded recursively from this parent folder - should mention this

Comment (SamMarkowitz) [inline cloudslang_cli.rst]: Is it the same for inputs files? Loaded recursively?

Comment (levice14) [inline cloudslang_cli.rst]: nope.. only for SP files (because they have namespace but inputs files are plain Yaml for now)

**Comment by levice14** [comment]
sam_wording

**Pull request merged**